celestia/peers.txt

```diff
@@ -1 +0,0 @@
-
```

Comment:
Can you simply not modify this file at all?

Reply:
@liamsi Hi, I just reverted and add cosmostation.txt